components/user-header.js

```diff
@@ -205,6 +205,7 @@ function HeaderHeader ({ user }) {
             className='me-1'
           />{user.name}@stacker.news
         </Button>
+        {user.isContributor && <div>Verified stacker.news contributor</div>}
```

Comment:
I am surprised this is styled like other `text-muted` and small text in the profile according to the screenshot since you only use a simple `<div>` here.

Reply:
Take a look at the later commits, this simple `div` was just the placeholder. I am using `small` with `.text-muted` like the other text in that area.

Reply:
Ah, that's what I get for reviewing immediately when I got the mail, lol